packages/storybook/src/generators/configuration/root-files/.storybook/main.js

```diff
@@ -1,4 +1,4 @@
 module.exports = {
   stories: [],
-  addons: ['@storybook/addon-knobs/register'],
+  addons: ['@storybook/addon-controls'],
```

Comment:
Here's an idea, maybe for the future:

Storybook has an addon called [essentials](https://storybook.js.org/docs/react/essentials/introduction). It's a bundle of curated addons by the maintainers with first world support, and that includes controls as well (and many super cool addons). So maybe instead of including controls, include essentials. The initial experience for the user will be relatively more interesting!

Reply:
So, if I add `@storybook/addon-essentials` then I can remove the `@storybook/addon-controls` package?

Reply:
definitely!